content/api/events/catalog-of-events.md

```diff
@@ -183,11 +183,11 @@ single test.
 
 #### To turn off all uncaught exception handling
 
-```javascript
-// likely want to do this in a support file
-// so it's applied to all spec files
-// cypress/support/index.js
+You'll likely want to do this in a
```

Comment:
Can you use `::include{file=partials/support-file-configuration.md}` instead of this content throughout this doc?